dear_petition/petition/migrations/0019_auto_20200407_1720.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.10 on 2020-04-07 16:49
+
+from django.db import migrations
+
+
+def update_existing_ciprs_records(apps, schema_editor):
+    CIPRSRecord = apps.get_model("petition", "CIPRSRecord")
+    records = CIPRSRecord.objects.all()
+    for r in records:
+        refresh_record_from_data(r)
+
+
+def refresh_record_from_data(record):
+    """Updates model fields that depends on the data JSONField
+
+    The record exists, but the raw data (data - JSONFIELD) has
+    changed. Let's update the models that are extracting data
+    from this field.
+    """
+    record.file_no = self.data["General"].get("File No", "")
```

Comment:
The `self` in here will need to be changed to `record`

Reply:
done